components/director/internal/domain/formationconstraint/operators/asynchronous_flow_control_operator_test.go

```diff
@@ -87,7 +87,7 @@ func TestConstraintOperators_AsynchronousFlowControlOperator(t *testing.T) {
 		},
 		// Assign during PreStatusReturned
 		{
-			Name:                             "Success when transitioning to READY state with inbound credentials",
+			Name:                             "Error when formation assignment config is invalid",
```

Comment:
Could we change the value of `invalidFAConfig` to `invalid-config` or something similar? Just to remove the destination specifics